aliases/available/general.aliases.bash

```diff
@@ -75,9 +80,34 @@ alias rd='rmdir'
 alias shit='bash-it'
 alias batshit='bash-it'
 alias bashit='bash-it'
+alias bsh='bash-it'
```

Comment:
Some operating systems have the Bash shell as `bsh`, which will be overwritten by this alias. I'm not in favor of adding this one.